#!/usr/bin/env python
PACKAGE = "my_pcl_tutorial"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("LEN_X", double_t, 0, "X length in meters",    .1, 0,   5)
gen.add("LEN_Y", double_t, 0, "Y length in meters",    .1, 0,   5)
gen.add("LEN_Z", double_t, 0, "Z length in meters",    .3, 0,   5)

gen.add("centre_X", double_t, 0, "X centre",    0, -3,   3)
gen.add("centre_Y", double_t, 0, "Y centre",    0, -3,   3)
gen.add("centre_Z", double_t, 0, "Z centre",    0, -3,   3)

gen.add("gain_X", double_t, 0, "X gain",    1, -1,   5)
gen.add("gain_Y", double_t, 0, "Y gain",    1, -1,   5)
gen.add("gain_Z", double_t, 0, "Z gain",    1, -1,  5)


exit(gen.generate(PACKAGE, "my_pcl_tutorial", "AutoCrop"))
